pkg/generate/app/cmd/describe.go

```diff
@@ -143,6 +143,9 @@ func describeBuildPipelineWithImage(out io.Writer, ref app.ComponentReference, p
 			fmt.Fprintf(out, "    * This image declares volumes and will default to use non-persistent, host-local storage.\n")
 			fmt.Fprintf(out, "      You can add persistent volumes later by running 'volume dc/%s --add ...'\n", pipeline.Deployment.Name)
 		}
+		if pipeline.Image.Info != nil && (len(pipeline.Image.Info.Config.User) == 0 || pipeline.Image.Info.Config.User == "root" || pipeline.Image.Info.Config.User == "0") {
```

Comment:
i'm going to remove the len==0 check, not sure why i put that in, as you say, it doesn't necessarily mean root.

and no, we don't want a warning for if they specified a specific(non-root) user.  I would assume (almost?) every image specifies a specific user, either root or something else.